src/mocks/CatalogueCategory.json

```diff
@@ -358,12 +376,14 @@
         "name": "Measurement Range",
         "type": "number",
         "unit": "volts",
-        "mandatory": true
+        "mandatory": true,
+        "allowed_values": null
       },
       {
         "name": "Accuracy",
         "type": "string",
```

Comment:
unit be return `null` if not given in `catalogue_item_properties`, instead of missing.